### ISSUE TITLE
Installation fails; get-json no longer available on NPM

### ISSUE THREAD

**Issue by bjacobel** [npm-flickr/flickr-client#1]
the `package.json` dependency `get-json` [no longer exists on npm](https://www.npmjs.com/package/get-json). For this reason, installation of this package via npm fails.

```
$ > npm install --save flickr-client
npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/Cellar/node/5.5.0/bin/node" "/usr/local/bin/npm" "install" "--save" "flickr-client"
npm ERR! node v5.5.0
npm ERR! npm  v3.5.3
npm ERR! code EREADFILE

npm ERR! Error extracting /Users/bjacobel/.npm/get-json/0.0.0/package.tgz archive: ENOENT: no such file or directory, open '/Users/bjacobel/.npm/get-json/0.0.0/package.tgz'
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/bjacobel/code/photos/npm-debug.log
```

**Comment by zeke** [comment]
Thanks for reporting, @bjacobel. I have filed an issue with npm to try to take over that package and get it republished.

**Comment by zeke** [comment]
I just published `0.0.3`. Try `npm i flickr-client` again and you should be good!
